src/storage/resourceManager.ts

```diff
@@ -52,28 +50,33 @@ type SubjectStringCache = Map<string, string[] | undefined>;
 type CoarseResourceType = KafkaClusterType | SchemaRegistryType | EnvironmentType;
 
 /**
- * Enumeration of strings describing coarse resource kinds, used to generate workspace storage keys.
+ * Enumeration of strings describing resource kinds we generate
+ * per-connection-id storage keys for.
+ *
  * See {@link ResourceManager.generateWorkspaceStorageKey}, and, say,
  * {@link ResourceManager.getEnvironments} / {@link ResourceManager.setEnvironments}.
  */
-export enum CoarseResourceKind {
+export enum GeneratedKeyResourceType {
```

Comment:
Renamed + added in that we handle the topics and subjects storage keys using the same strategy, now all connectionid-resourcetype.